includes/class-wc-calypso-bridge-woocommerce-admin.php

```diff
@@ -37,6 +37,7 @@ public function init() {
 		add_filter( 'wc_admin_get_feature_config', array( $this, 'maybe_remove_devdocs_menu_item' ) );
 		add_filter( 'pre_option_woocommerce_task_list_hidden', array( $this, 'disable_new_task_list' ) );
 		add_filter( 'pre_option_woocommerce_onboarding_opt_in', array( $this, 'disable_onboarding_opt_in' ) );
+		add_filter( 'pre_option_woocommerce_setup_ab_wc_admin_onboarding', array( $this, 'disable_onboarding_a_b_test' ) );
```

Comment:
If we don't have the `new_onboarding` step we hopefully won't hit this check at all.

https://github.com/woocommerce/woocommerce/blob/6a8d8b27d98fe3036f2240d14be2b7a5e4ecf10e/includes/admin/class-wc-admin-setup-wizard.php#L278